examples/generative/corrdiff/training/training_loop.py

```diff
@@ -231,11 +237,11 @@ def training_loop(
                     labels=labels,
                     augment_pipe=augment_pipe,
                 )
-                training_stats.report("Loss/loss", loss)
+                training_stats.report("Loss/training loss", loss)
```

Comment:
this name change will break some people loss plotting script, but I believe we should make this change to avoid ambiguity

Reply:
maybe keep the name the same, and but use a new name for the validation loss.